### PR TITLE
feat: don't upload snapshots to DigitalOcean

### PR DESCRIPTION
**Pull request by lemmih**
**Summary of changes**
Changes introduced in this pull request:
- Don't upload files to DigitalOcean.
- Don't prune files from DigitalOcean.

Pruning snapshot files in CloudFlare is done separately.

**Reference issue to close (if applicable)**

Closes

**Other information and links**

### CODE REVIEW

File: terraform/modules/daily_snapshot/service/snapshots_prune.rb
Comment (LesnyRumcajs) [inline terraform/modules/daily_snapshot/service/snapshots_prune.rb:1]: Let's remove the tests as well https://github.com/ChainSafe/forest-iac/tree/main/terraform/modules/daily_snapshot/service/tests

**Comment by LesnyRumcajs** [comment]
Let's remove https://github.com/ChainSafe/forest-iac/blob/main/.github/workflows/tests-daily-snapshot.yml

Apparently they were not working for some time due to code re-organization in the repository.

**Comment by lemmih** [comment]
> Let's remove https://github.com/ChainSafe/forest-iac/blob/main/.github/workflows/tests-daily-snapshot.yml
> 
> Apparently they were not working for some time due to code re-organization in the repository.

Done. So much code that I didn't know about. :)

File: terraform/modules/daily_snapshot/service/daily_snapshot.rb [hunk 1]
```diff
@@ -17,6 +15,19 @@
 BUCKET = get_and_assert_env_variable 'SNAPSHOT_BUCKET'
 ENDPOINT = get_and_assert_env_variable 'SNAPSHOT_ENDPOINT'
 
+# Query the date of the most recent snapshot.
+def latest_snapshot_date(chain_name = 'calibnet')
+  # We do not support HEAD requests but we _do_ support empty ranges.
+  filename = `curl --remote-name --remote-header-name --write-out "%{filename_effective}" --silent https://forest-archive.chainsafe.dev/latest/#{chain_name}/ -H "Range: bytes=0-0"`
+  # Curl will create a file with a single byte in it. Let's clean it up.
```
Comment (LesnyRumcajs) [inline terraform/modules/daily_snapshot/service/daily_snapshot.rb:22]: could we not save it to a file instead?

**Review by LesnyRumcajs — APPROVED** [review]

File: terraform/modules/daily_snapshot/service/daily_snapshot.rb (hunk 1 above)
Comment (lemmih) [reply terraform/modules/daily_snapshot/service/daily_snapshot.rb:22]: I'd rather not deal with file management in Ruby. Forest is much better at it.

Comment (lemmih) [reply terraform/modules/daily_snapshot/service/daily_snapshot.rb:22]: And we _do_ need a way to get the date _without_ downloading the file in any case.

**Pull request merged**